README.md

```diff
@@ -19,9 +19,10 @@ This project can be cloned and re-configured to fit your needs but we highly sug
 Here's how this example project was created:
 
 1. Create a new project directory: `$ mkdir example.com && cd example.com`
-2. Clone Trellis: `$ git clone --depth=1 git@github.com:roots/trellis.git && rm -rf trellis/.git`
+2. Clone Trellis: `$ git clone --depth=1 git@github.com:roots/trellis.git trellis && rm -rf trellis/.git`
```

Comment:
this was done on purpose [when we renamed dirs](https://github.com/roots/roots-example-project.com/commit/f108d88037b590b18c1072207e0afc43b750c7e2#diff-04c6e90faac2675aa89e2176d2eec7d8R22) — trellis is already going to be the name of the directory upon running this command